app/customer.py

```diff
@@ -0,0 +1,151 @@
+import json
+from app.shop import Shop
+from datetime import datetime
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list,
+            money: int,
+            car: dict
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+        self.gas_prices = {
+            "Outskirts Shop": 0,
+            "Shop '24/7'": 0,
+            "Central Shop": 0
+        }
+        self.cart_cost = {
+            "Outskirts Shop": {
+                "milk": 0,
+                "bread": 0,
+                "butter": 0
+            },
+            "Shop '24/7'": {
+                "milk": 0,
+                "bread": 0,
+                "butter": 0
+            },
+            "Central Shop": {
+                "milk": 0,
+                "bread": 0,
+                "butter": 0
+            }
+        }
+        self.gas_cart = {
+            "Outskirts Shop": 0, "Shop '24/7'": 0, "Central Shop": 0
+        }
```

Comment:
Hardcoded data. If customer want buy a cake? And also magazine will have a cake? What will you do? Change code? Remove hardcoded values. Use loops!